setup/config/config_clientapi.go

```diff
@@ -93,6 +90,10 @@ func (c *ClientAPI) Verify(configErrs *ConfigErrors, isMonolith bool) {
 		if c.RecaptchaSitekeyClass == "" {
 			c.RecaptchaSitekeyClass = "g-recaptcha-response"
 		}
+		checkNotEmpty(configErrs, "client_api.recaptcha_public_key", c.RecaptchaPublicKey)
```

Comment:
Why shift the order of checks here?

Reply:
Could remove it completely, given we either have something set or default to reCaptcha (if nothing is set).